### DISCUSSION TITLE
Ar67

### DISCUSSION THREAD

**Discussion by ar-67** [arkypita/LaserGRBL#2180] [Q&A]
He cambiado el láser a más potente pero el lasergrbl no lo enciende,como tengo que hacerlo?
Gracias

**Comment by Tribo-2dot0** [comment]
Para LaserGrbl não importa se o laser mudou para um mais potente, se ainda $30=1000 então 1000 ainda será 100%... eu verificaria novamente as conexões dos cabos.
